cdap-common/src/main/java/co/cask/cdap/common/utils/TimeMathParser.java

```diff
@@ -80,19 +81,27 @@ public static int resolutionInSeconds(String resolutionStr) {
   }
 
   public static long parseTime(String timeStr) {
```

Comment:
All public methods should have javadoc. I don't know why the existing code get that slipped. Anyway, when seeing it is missing, please add it.